fedmsg_meta_fedora_infrastructure/tests/pagure.py

```diff
@@ -1555,6 +1555,75 @@ class TestPullRequestFlagUpdated(Base):
         }
 
 
+class TestGitCommit(Base):
+    """ These messages are published when a someone updates a flag on a
+    pull-request on a project on `pagure <https://pagure.io>`_.
+    """
+    expected_title = "pagure.git.receive"
+    expected_subti = 'pingou@fp.o pushed to pagure (bleach). '\
+        '"Use the default attributes for bleach and then add our own"'
+    expected_link = "https://pagure.io/pagure/131e07ed2538839d509880a011cd7d55c0967171"
+    expected_icon = "https://apps.fedoraproject.org/packages/" + \
+        "images/icons/package_128x128.png"
+    expected_secondary_icon = None
+    expected_packages = set([])
+    expected_usernames = set()
```

Comment:
Should `pingou` be returned in the usernames here?  If you change ``pingou@fp.o`` here to ``pingou@fedoraproject.org`` and then use the ``email2fas`` utility, you could do it.